theme/accessibility.less

```diff
@@ -8,24 +8,28 @@
 /*******************************
           Accessibility
 *******************************/
-
-*:focus {
-    outline: 1px solid transparent !important;
+*[tabindex='0']:focus,
+*[tabindex*='d1']:focus,
+*[tabindex*='d2']:focus, /* Takes items with a defined tabIndex from 0 to 29. */
+*[role='menuitem']:focus {
+    outline: 2px solid #4D90FE !important;
```

Comment:
Does this affect Blockly or Monaco?

Reply:
No it doesn't. Below, there is a CSS block that remove the focus outline on Blockly and Monaco.